packages/kit-headless/src/components/combobox/combobox-input.tsx

```diff
@@ -18,7 +18,7 @@ import {
   getPrevEnabledOptionIndex,
 } from './utils';
 
-const preventedKeys = [KeyCode.Home, KeyCode.End, KeyCode.PageDown, KeyCode.ArrowUp];
+const comboPreventedKeys = [KeyCode.Home, KeyCode.End, KeyCode.PageDown, KeyCode.ArrowUp];
```

Comment:
do we need to differentiate between combo and list?

Reply:
This fixed an error `Conflicting namespaces: "packages/kit-headless/src/index.ts" re-exports "_auto_preventedKeys" from one of the modules "packages/kit-headless/src/components/autocomplete/index.ts" and  "packages/kit-headless/src/components/combobox/index.ts" (will be ignored)`

I'm fine not fixing this at this time as well b/c I did not see any actual issue stemming from it. 

What do you think?

Reply:
Well I don't see this error when I build the docs website, where do you see it?

Let's open a different issue / PR for this error 
I'll remove it from this PR for now just to merge it 
but we can easily rename it if needed